tests/test_generic/tests/test_mandatory_files.py

```diff
@@ -17,11 +19,20 @@ class FileTest(IndustryCase):
 
     def test_required_files(self):
         for module in self.installed_modules:
+            needs_translation = False
+            with io.BytesIO() as buffer:
+                trans_export(False, [module], buffer, 'po', self.env)
+                if "model_terms:" in buffer.getvalue().decode("utf-8"):
```

Comment:
Does not work for all. Maybe this is more generic
```suggestion
                if f"module: {module}" in buffer.getvalue().decode("utf-8"):
```